nilrt_snac/_configs/_console_config.py

```diff
@@ -11,14 +12,16 @@ def __init__(self):
         pass  # Nothing to do for now
 
     def configure(self, args: argparse.Namespace) -> None:
-        print("Configuring console access...")
-        dry_run: bool = args.dry_run
-        if not dry_run:
-            logger.debug("Disabling console access")
-            subprocess.run(
-                ["nirtcfg", "--set", "section=systemsettings,token=consoleout.enabled,value=False"],
-                check=True,
-            )
+        print("Deconfiguring console access...")
+
+        if args.dry_run:
+            return
```

Comment:
Returning early like this affects logging, since the `opkg_helper` methods print stuff. I don't know if we care about that, though, so feel free to disregard this.

Reply:
I think that's OK. As a user, I wouldn't expect logging to be consistent between dry-run and wet-run modes of operation, as long as the general steps of operation are consistent - which they are.

Reply:
> I wouldn't expect logging to be consistent between dry-run and wet-run modes

I wasn't thinking about consistency between those modes as much as consistency between the different config plugins. For example, `_NIAuthConfig` does not return early, so it will execute `opkg_helper`'s prints in a dry run whereas `_ConsoleConfig` will not. Anyway, again, feel free to disregard.

Reply:
Hmm... it looks more like the NIAuth Config just doesn't honor the dry-run setting at all. It looks like it's supposed to [`set_dry_run()`](https://github.com/ni/nilrt-snac/blob/master/nilrt_snac/opkg.py#L39), but doesn't. So I think [those `remove()` lines](https://github.com/ni/nilrt-snac/blob/master/nilrt_snac/_configs/_niauth_config.py#L17) always run. :/

Reply:
Filed bug #46